src/operators/rasterization.md

```diff
@@ -0,0 +1,112 @@
+# Rasterization
+
+The `Rasterization` operator creates a raster from a point vector source.
+It offers two options for rasterization: A grid rasterization and a (gaussian) density rasterization (heatmap).
+
+## Inputs
+
+The `Rasterization` operator expects exactly one _vector_ input.
+
+| Parameter | Type                 |
+|-----------|----------------------|
+| `source`  | `SingleVectorSource` |
+
+## Parameters
+### Grid Rasterization
+
+| Parameter           | Type                | Description                                                | Example Value            |
```

Comment:
The `type` is missing here

Reply:
https://github.com/geo-engine/docs/pull/35/commits/4341cd366bfe287effb41f6b74fa40a45ccbda6e